nixos/modules/services/torrent/opentracker.nix

```diff
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.opentracker;
+
+  out = {
+    options.services.opentracker = api;
+    config = lib.mkIf cfg.enable imp;
+  };
+
+  api = {
+    enable = mkEnableOption "opentracker";
+
+    package = mkOption {
+      type = types.package;
+      description = ''
+        opentracker package to use
+      '';
+      default = pkgs.opentracker;
+    };
+
+    extraOptions = mkOption {
+      type = types.separatedString " ";
+      description = ''
+        Configuration Arguments for opentracker
+        See https://erdgeist.org/arts/software/opentracker/ for all params
+      '';
+      default = "";
+    };
+
+    user = mkOption {
+      type = types.str;
+      description = ''
+        User which will run opentracker. by default opentracker drops all
+        privileges and runs in chroot after starting up as root.
```

Comment:
This comment indicates to me that opentracker does chrooting/privsep by itself, but that won't work with `serviceConfig.User`. But if it doesn't do itself, the isolation options used below are not comparable to an ordinary chroot.